README.md

```diff
@@ -13,6 +13,56 @@ Namely:
 
 Run `make install`
 
+## How to use custom CSS stylesheet
+
+You can use a custom CSS stylesheet to change a look of workspace indicator widget.
+
+#### 1. Write your own CSS stylesheet
+
+As a base, use [stylesheet.css](./stylesheet.css), as it has all classes needed to create a custom stylesheet for IWI.
+
+> **Note**
+> Creating custom CSS stylesheets might require from you some actual knowledge of CSS. You can check [this](https://developer.mozilla.org/en-US/docs/Learn/CSS/First_steps) tutorial on MDN to learn basics of CSS.
+
+<details open>
+<summary>Which parts of a stylesheet you would want to modify:</summary>
+
+#### Workspace indicator
+```css
+.panel-workspace-indicator,
+.panel-workspace-indicator-box .workspace {
+  border: 1px solid #cccccc;
+  background-color: rgba(200, 200, 200, 0.5);
+}
+```
+###### Located in lines [13-17](https://github.com/MichaelAquilina/improved-workspace-indicator/blob/b03afe9d3fe562c418ff25967e61eded67bf17c6/stylesheet.css#L13-L17)
```

Comment:
these might quickly break over time but I guess we can see how it goes and take a different approach in the future if needed

Reply:
If it ever becomes a problem, I can always help in fixing it. I'm already subscribed to new PRs, so I'll keep an eye on new features added to IWI 😉️